lightning/src/ln/channelmanager.rs

```diff
@@ -1299,6 +1303,10 @@ pub struct ChannelDetails {
 	pub is_usable: bool,
 	/// True if this channel is (or will be) publicly-announced.
 	pub is_public: bool,
+	/// The smallest value HTLC (in msat) we will accept, for this channel.
+	pub htlc_minimum_msat: u64,
```

Comment:
Maybe lets call these "inbound" or something (like "inbound_htlc_minimum_msat"), to make it clearer without needing to look at docs what they're talking about.

Reply:
Will do, thanks!